lib/iris/tests/test_interpolation.py

```diff
@@ -510,25 +510,6 @@ def test_points_datatype_casting(self):
         # this test tries to extract a float from an array of type integer. the result should be of type float.
         r = iris.analysis.interpolate.linear(self.simple2d_cube_extended, [('shared_x_coord', 7.5)])
         self.assertCML(r, ('analysis', 'interpolation', 'linear', 'simple_casting_datatype.cml'))
-        
```

Comment:
I understand the sentiment, I would just like to understand why these are now obsolete.  please could you expand a little on why these are not necessary and which remaining tests provide the test coverage?
thank you